components/filter/filter-dimension-set.js

```diff
@@ -11,6 +11,11 @@ class FilterDimensionSet extends LitElement {
 
 	static get properties() {
 		return {
+			/**
+			 * The header text to display as the list header
```

Comment:
Maybe more of a Jeff question, but do we want to give a bit of insight here on when to use this? Similar to the introductory test, it takes up space and shouldn't really be needed unless you need to provide that extra context because searching will provide different options?

Reply:
Yeah, it would be good to add some additional context. The only use case of the header text at this point is for when the filter displays different lists of content when searching and not searching. Though I'm unsure if this is the only case this should be used. I'll check with Jeff.

Reply:
Jeff and I came up with an update for the description. It was difficult to generalize but we didn't want to specify it as only being for the case that Wave is using it for.

We could have added some additional insight on when to use it to the documentation page for the component but we thought that may encourage more users to use the property, which we don't want, so opted for the bulky and slightly vague description.